.github/workflows/run-tests.yml

```diff
@@ -2,17 +2,15 @@ name: Run tests
 
 on:
   pull_request:
-    branches: [main]
-  push:
-    branches: [main]
+    branches: [1.29]
 
 jobs:
   run-tests:
     name: Run tests
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        channel: [latest/edge, latest/edge/strict]
+        channel: [1.29/edge, 1.29-strict/edge]
 
```

Comment:
that would be just for 1.29/edge, no? the `1.29-strict` branch will handle the runs for strict

Reply:
resolved offline: we do not maintain a separate strict branch for the addon repository.